File: get_apikey.js
const axios = require('axios');
const base_url = 'https://solve.shimul.me'

const user_id='test'
const email='test@example.com'
const password = 'password'
const name = 'Jhon Doe'


const result = axios({
  method: 'post',
  url: base_url+'/account/signup',
  headers: {
  'Content-type': 'application/json'
  },
  data: data={"user_id": user_id, "email": email, "password": password, "name": name}
})
 .then((response) => {
    console.log(response.status);
    console.log(response.data)
})
.catch((error) => {
    console.log(error);
  });

File: get_apikey.py
# pip install requests
import requests, json
base_url = 'https://solve.shimul.me'

user_id='test'
email='test@example.com'
password = 'password'
name = 'Jhon Doe'

r = requests.get(url = base_url+'/account/signup', headers={'Content-Type': 'application/json'}, data = json.dumps({'user_id': user_id, 'email': email, 'password': password, 'name': name}))

print(json.dumps(r.json()))